frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -12,14 +12,44 @@ import {
   applicationName,
   appName,
   createButton,
+  setRouting,
+  setBuildConfig,
+  setDeployment,
+  setResources,
+  setLabel,
+  selectAdvancedOptions,
+  AdvancedOptions,
+  TLSTerminationValues,
+  buildConfigObj,
+  addNavigate,
+  routingObj,
+  deploymentObj,
+  resourceLimitsObj,
+  setScaling,
+  scalingObj,
+  setSecureRoute,
+  pipelineObj,
+  setPipelineForGitFlow,
+  importFromGitHeader,
+  addApplicationInGeneral,
+  gitUrlHelper,
+  builderImageSelected,
 } from '../views/git-import-flow.view';
+import { 
+  // verifyCheckBox, 
+  enterText} from '../utilities/elementInteractions';
 import { newApplicationName, newAppName } from '../views/new-app-name.view';
 import { switchPerspective, Perspective, sideHeader } from '../views/dev-perspective.view';
+import {pipelinecheckStatus, pipelineTableBody} from '../views/pipeline.view';
+import {elementByDataTestID, click} from '../utilities/elementInteractions';
+const waitForElement = 5000;
```

Comment:
move this const after import statements

Reply:
Is it a part of coding practice ?? If Yes,  I can move this line